src/generated/resources/openapi.json

```diff
@@ -1598,19 +1747,19 @@
       "HealthCheck" : {
         "type" : "object",
         "properties" : {
-          "status" : {
-            "enum" : [ "UP", "DOWN" ],
-            "type" : "string"
-          },
           "data" : {
             "type" : "object",
             "nullable" : true
           },
+          "status" : {
+            "enum" : [ "UP", "DOWN" ],
+            "type" : "string"
+          },
```

Comment:
Do you mind removing this unrelated change from the PR?

Reply:
Removed, thanks!